string.c

```diff
@@ -11525,6 +11525,260 @@ rb_enc_interned_str_cstr(const char *ptr, rb_encoding *enc)
+ *  - #unicode_normalize:: Returns a copy of +self+ with each character Unicode-normalized.
+ *  - #encode:: Returns a copy of +self+ with all characters transcoded from one given encoding into another.
+ *
+ *  _Casing_
+ *
+ *  - #capitalize:: Returns a copy of +self+ with the first character upcased
+ *                  and all other characters downcased.
+ *  - #downcase:: Returns a copy of +self+ with all characters downcased.
+ *  - #upcase:: Returns a copy of +self+ with all characters upcased.
+ *  - #swapcase:: Returns a copy of +self+ with all upcase characters downcased
+ *                and all downcase characters upcased.
+ *
+ *  _Deletion_
+ *
+ *  - #delete:: Returns a copy of +self+ with characters removed
+ *  - #delete_prefix:: Returns a copy of +self+ with a given prefix removed.
+ *  - #delete_suffix:: Returns a copy of +self+ with a given suffix removed.
+ *  - #lstrip:: Returns a copy of +self+ with leading whitespace removed.
+ *  - #rstrip:: Returns a copy of +self+ with trailing whitespace removed.
+ *  - #strip:: Returns a copy of +self+ with leading and trailing whitespace removed.
+ *  - #[]=:: Returns a substring o +self+ determined by given index, range, regexp, or string.
+ *  - #chomp:: Returns a copy of +self+ with a trailing record separator removed, if found.
+ *  - #chop:: Returns a copy of +self+ with trailing whitespace or the last character removed.
+ *  - #squeeze:: Returns a copy of +self+ with contiguous duplicate characters removed.
+ *  - #[], #slice:: Returns a substring determined by a given index, start/length, or range, or string.
+ *  - #byteslice:: Returns a substring determined by a given index, start/length, or range.
+ *  - #chr:: Returns the first character.
+ *
+ *  _Duplication_
+ *
+ *  - #to_s:: If +self+ is a subclass of \String, returns +self+ copied into a \String;
+ *            otherwise, returns +self+.
+ *  - #to_str:: If +self+ is a subclass of \String, returns +self+ copied into a \String;
+ *              otherwise, returns +self+.
+ *
+ *  === Methods for Converting to Non-\String
+ *
+ *  Each of these methods converts the contents of +self+ to a non-\String.
+ *
+ *  <em>Characters and Bytes</em>
+ *
+ *  - #bytes:: Returns an array of the bytes in +self+.
+ *  - #chars:: Returns an array of the characters in +self+.
+ *  - #codepoints:: Returns an array of the integer ordinals in +self+.
+ *  - #getbyte:: Returns an integer byte as determined by a given index.
+ *
+ *  _Splitting_
+ *
+ *  - #lines:: Returns an array of the lines in +self+, as determined by a given record separator.
+ *  - #partition:: Returns a 3-element array determined by the first substring that matches
+ *                 a given substring or regexp,
+ *  - #rpartition:: Returns a 3-element array determined by the last substring that matches
+ *                  a given substring or regexp,
+ *  - #split, #to_a:: Returns an array of substrings determined by a given delimiter -- regexp or string --
```

Comment:
`to_a` is not a String method, and should be removed.

Reply:
I did some investigation to determine which are true String methods and which not.  Is there a reliable way to determine?

Reply:
Fixed.

Reply:
If you want to know what is actually a string method, open up irb and see if the method call works.  You can also look at the method definitions in `string.c` (search for the method name in double quotes).